staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -345,9 +351,10 @@ func insert(q *waitForPriorityQueue, knownEntries map[t]*waitFor, entry *waitFor
 			heap.Fix(q, existing.index)
 		}
 
-		return
+		return false
 	}
 
 	heap.Push(q, entry)
 	knownEntries[entry.data] = entry
+	return true
```

Comment:
why don't just add the metric here so you don't have to modify the functions?

Reply:
👍 Changed `insert` to be a method instead so I'm able to set the metric here.